pkg/statdb/sdbclient/client.go

```diff
@@ -136,6 +138,28 @@ func (sdb *StatDB) Update(ctx context.Context, nodeID []byte, auditSuccess, isUp
 	return res.Stats, err
 }
 
+// UpdateUptime is used for updating a node's uptime in statdb
+func (sdb *StatDB) UpdateUptime(ctx context.Context, nodeID []byte,
+	isUp bool) (stats *pb.NodeStats, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	node := pb.Node{
+		NodeId:             nodeID,
+		IsUp:               isUp,
+		UpdateAuditSuccess: false,
+		UpdateUptime:       true,
+		UpdateLatency:      false,
+	}
+	updateReq := &pb.UpdateRequest{
+		Node:   &node,
+		APIKey: sdb.APIKey,
+	}
+
+	res, err := sdb.client.Update(ctx, updateReq)
```

Comment:
needs `if err != nil` bc this will panic if res is ever nil

Reply:
yep, good catch haha